packages/wmr/src/lib/prerender.js

```diff
@@ -131,7 +131,7 @@ async function workerCode({ cwd, out, publicPath }) {
 		if (result.links) {
 			for (let url of result.links) {
 				const parsed = new URL(url, 'http://localhost');
-				url = parsed.pathname + parsed.search;
+				url = parsed.pathname;
```

Comment:
Wondering why we had this in the first place? 🤔 did we had a use case?

Reply:
Perhaps the idea was that query params can by definition result in a completely different page (e.g. web API endpoint /get?id=about vs. /get?id=home), compared with the hash fragment part of the URL which has different semantics in a traditional non-SPA client/server context (i.e. not used for Single Page Application routing).

Reply:
Correct!